spec/spec_helper.rb

```diff
@@ -50,8 +51,26 @@ def require_relative(path)
   end
 
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
-  config.use_transactional_fixtures = true
-  config.use_instantiated_fixtures  = false
+
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.strategy = :transaction
+  end
+
+  config.before(:each, :type => :worker) do
+    DatabaseCleaner.strategy = :truncation
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.start
+  end
+
+  config.append_after(:each) do
+    DatabaseCleaner.clean
```

Comment:
I assume this uses the transaction strategy to be as similar to the use_transactional_fixtures way for non-workers... (same question as above)